src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/HTMLCSharpLanguageServerFeedbackFileLoggerProvider.cs

```diff
@@ -27,7 +32,7 @@ internal class HTMLCSharpLanguageServerFeedbackFileLoggerProvider : ILoggerProvi
             _loggerProvider = (FeedbackFileLoggerProvider)loggerFactory.GetOrCreate(LogFileIdentifier);
         }
 
-        public ILogger CreateLogger(string categoryName) => _loggerProvider.CreateLogger(categoryName);
+        public virtual ILogger CreateLogger(string categoryName) => _loggerProvider.CreateLogger(categoryName);
```

Comment:
Leave a comment stating that it's only virtual for  testing

Reply:
```suggestion
        // Virtual for testing
        public virtual ILogger CreateLogger(string categoryName) => _loggerProvider.CreateLogger(categoryName);
```